dockerfiles/Dockerfile

```diff
@@ -31,6 +31,8 @@ FROM $BASE_IMAGE
 MAINTAINER Sam Ritchie <sritchie09@gmail.com>
 
 ARG GCLOUD_LOC=/usr/local/gcloud
+ARG PYTHON_VERSION=3.8
+ARG MINICONDA_VERSION=py38_4.8.2
```

Comment:
what versions do they support? Is it every cross of python version and major-minor-incremental? Can you add a link here to the page that lists their versions? this sort of string hacking is tricky, I hear you.

Reply:
Not sure on this, looking at the conda docs, I would guess that they can support versions < that reflected in miniconda release tag, but that's just a guess. I'll add a link to the docs and the miniconda release archive here.